### PR TITLE
[CodeSec AG] Add supported languages for IaC scans [RLP-86594]

### PR DESCRIPTION
**Pull request by anaghapamidi**
Summary of changes:

Modified files:
- book.yml

Added files:
- supported-languages-for-specific-domains.adoc

### CODE REVIEW

**Review by Pubs-MV — COMMENTED** [review]
Please check with @tsmithv11 on the best place to add this. I think we need a page for IaC Security, and this can be a part of that.

**Comment by anaghapamidi** [comment]
@tsmithv11 - Request your review here.

**Comment by Pubs-MV** [comment]
@tsmithv11 , I wanted to confirm if this should be a standalone reference or if it part of a topic on Iac Scan that includes a section on supported languages and frameworks. 

Now that we have a Iac Scan module, it seems like we need a topic on it that says what it is and links to other sections of the guide as appropriate. Please guide on what we need to say and whether we should rename this topic (and filename) for Iac Scan .

File: code-security/admin_guide/book.yml [hunk 1]
```diff
@@ -93,6 +93,8 @@ topics:
     file: add-terraform-enterprise-run-tasks.adoc
   - name: Connect VScode with Prisma Cloud Code Security
     file: connect-vscode.adoc
+- name: Supported Languages for Specific Domains on Prisma Cloud Code Security
```
Comment (tsmithv11) [inline code-security/admin_guide/book.yml:96]: 2 things:

1. Why not have a new section for IaC like we do for SCA? https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/software-composition-analysis/supported-package-managers
2. Let's make this more searchable. How about `Supported IaC and CI/CD languages and frameworks`

**Comment by tsmithv11** [comment]
> Please check with @tsmithv11 on the best place to add this. I think we need a page for IaC Security, and this can be a part of that.

Great minds

File: code-security/admin_guide/book.yml (hunk 1 above)
Comment (anaghapamidi) [reply code-security/admin_guide/book.yml:96]: @tsmithv11 - the placement of this file looks fine here? Can rename the file.

cc: @Pubs-MV

Comment (tsmithv11) [reply code-security/admin_guide/book.yml:96]: Yeah, it works for now. I'm saying in the future, we should consider an IaC page that includes the DSLs among a description.

**Pull request merged**